### PR TITLE
Update yml settings view

### PR DESCRIPTION
**Pull request by weban**

### CODE REVIEW

**Comment by weban** [comment]
@Nat-ess here is the redesign of yml configs. Please take a look

File: app/components/job-tabs.js [hunk 1]
```diff
@@ -11,5 +12,11 @@ export default Component.extend({
     if (isEmpty(this.$('.travistab-nav--secondary').find('.active'))) {
       this.$('#tab_log').addClass('active');
     }
-  }
+  },
+
+  tabTitle: computed('job.build.request.noYaml', function () {
+    if (this.get('job.build.request.noYaml')) { return null; }
+
+    return 'Look at this job’s config';
```
Comment (clekstro) [inline app/components/job-tabs.js]: Maybe `View this job's config` would be better?

File: app/components/job-tabs.js [hunk 2]
```diff
@@ -11,5 +12,11 @@ export default Component.extend({
     if (isEmpty(this.$('.travistab-nav--secondary').find('.active'))) {
       this.$('#tab_log').addClass('active');
     }
-  }
+  },
+
+  tabTitle: computed('job.build.request.noYaml', function () {
+    if (this.get('job.build.request.noYaml')) { return null; }
```
Comment (artursmirnov) [inline app/components/job-tabs.js]: It seems like `job.build.request.noYaml` worth an alias, as it already has multiple occurrences in template and now in component too.
```js
noYaml: reads('job.build.request.noYaml')
```

In this case this computed property could be simplified as well:
```js
tabTitle: computed('noYaml', function () {
  return this.noYaml ? '' : 'View this job's config';
})
```

**Comment by artursmirnov** [comment]
Also tests are failing.

**Comment by weban** [comment]
@artursmirnov pelase take a look again

**Comment by artursmirnov** [comment]
Hey @weban could you please resolve conflicts on the PR and fix the https://github.com/travis-ci/travis-web/pull/2035#discussion_r297562996 comment.

**Pull request closed without merging**